sdk/core/System.ClientModel/src/ModelReaderWriter/ModelReaderWriterOptions.cs

```diff
@@ -33,4 +38,115 @@ public ModelReaderWriterOptions (string format)
     /// Gets the format to read and write the model.
     /// </summary>
     public string Format { get; }
+
+    /// <summary>
+    /// Registers an <see cref="IPersistableModel{T}"/> proxy to be used when reading or writing a model.
```

Comment:
nit for clarity:

```suggestion
    /// Registers an <see cref="IPersistableModel{T}"/> proxy model that the ModelReaderWriter or model graph will use reading or writing a model of type <code>T</code>.
```

Reply:
I think its clear that since this is the ModelReaderWriterOptions the ModelReaderWriter will use it, I am not sure what you mean by "model graph" will use, this is not a concept in the MRW apis.

Reply:
But it will not be used in 100% of cases, correct?  For example, if a caller calls model.Write, it will not use a proxy?  

Given this I think it would be more clear to specify when it will be used in the public refdoc comment.